packages/client/src/generation/TSClient/PrismaClient.ts

```diff
@@ -106,8 +106,8 @@ function clientTypeMapOthersDefinition(this: PrismaClientClass) {
   })
 
   const argsResultMap = {
-    $executeRaw: { args: '[query: TemplateStringsArray | Prisma.Sql, ...values: any[]]', result: 'any' },
-    $queryRaw: { args: '[query: TemplateStringsArray | Prisma.Sql, ...values: any[]]', result: 'any' },
```

Comment:
So this is only a change impacting clientExtensions? The function name is a bit unclear

Reply:
Yes, that's for extensions-only